src/main/util/data_util.py

```diff
@@ -1,13 +1,13 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
 import os
-from typing import Any, Union
-
 import pandas as pd
+from typing import Any, Union, Callable
```

Comment:
`from typing import Any, Union, Callable`
should be before 
`import pandas as pd`
according to PEP8